public/apps/configuration/panels/permission-list/permission-list.tsx

```diff
@@ -209,18 +217,42 @@ export function PermissionList(props: AppDependencies) {
     </EuiContextMenuItem>,
   ];
 
+  const showEditModal = (
+    initialGroupName: string,
+    action: Action,
+    initialAllowedAction: string[]
+  ) => {
+    setEditModal(
+      <PermissionEditModal
+        groupName={initialGroupName}
+        action={action}
+        allowedActions={initialAllowedAction}
+        optionUniverse={actionGroups.map((group) => stringToComboBoxOption(group.name))}
+        handleClose={() => setEditModal(null)}
```

Comment:
Minor: from EUI doc, visibility is used for control. Just curious why not use same.

  const closeModal = () => setIsModalVisible(false);

  const showModal = () => setIsModalVisible(true);

Reply:
`const showModal = () => setIsModalVisible(true);` 
Doing that way we need to keep the states on the modal (updated group name and children permissions) at this level. I was trying to push down those to modal component level. Though not sure which way is better...